### PR TITLE
Add integration tests for splitter

### PR DESCRIPTION
**Pull request by stuartellis**
Docker integration tests.

### CODE REVIEW

File: integration/acceptance/splitter_spec.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# encoding: utf-8
```
Comment (tombruijn) [inline integration/acceptance/splitter_spec.rb]: don't think this is necessary anymore? Strings are using the UTF-8 encoding by default.

File: integration/acceptance/splitter_spec.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+require File.expand_path("../../spec_helper", __FILE__)
+
+module Backup
+  describe Splitter do
```
Comment (tombruijn) [inline integration/acceptance/splitter_spec.rb]: the nesting within the `Backup` module is something that has bothered me with other specs, it's not really necessary. We can also just write this directly:

```ruby
describe Backup::Splitter do
```

File: integration/acceptance/splitter_spec.rb (hunk 1 above)
Comment (stuartellis) [reply integration/acceptance/splitter_spec.rb]: Good point - this application is effectively Ruby 2 only, so these aren't necessary. I will take them out.

File: integration/acceptance/splitter_spec.rb (hunk 2 above)
Comment (stuartellis) [reply integration/acceptance/splitter_spec.rb]: I'll update this PR, and do another to clean up the existing files in the integration/ directory.

Thank you for reviewing!

**Pull request merged**